index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"> 
+    <title>My Portolio Website</title>
+</head>
+<body>
+    <main class="wrapper">
+        <header class="menu">
+            <div class="welcome-logo">
+                <a href="#">Welcome</a>
+            </div>
+            <nav class="menu-button">
+                <ul>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                </ul>
+            </nav>
+        </header>
+    </main>
+    <section class="hero">
+        <h2 class="hidden">Hero</h2>
```

Comment:
- [ ] Great job with the headings, however, this line is unnecessary, it would be better to remove it as this would make yyour code cleaner.